content/articles/netlify-service.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Netlify Service
```

Comment:
deprecated but we still offer it as an option in the OcS so I'm updating it to match the others.